core/templates/dev/head/pages/exploration_editor/translation_tab/AudioTranslationBarDirective.ts

```diff
@@ -225,13 +233,50 @@ oppia.directive('audioTranslationBar', [
             $scope.showRecorderWarning = false;
           };
 
+          $scope.stopRecording = function() {
+            $scope.voiceoverRecorder.stopRecord();
+            $scope.recordingComplete = true;
+            cancelTimer();
+            $scope.voiceoverRecorder.getMp3Data().then(function(audio) {
+              var fileType = 'audio/mp3';
+              $scope.audioBlob = new Blob(audio, {type: fileType});
+              // free the browser from workers
+              $scope.voiceoverRecorder.closeRecorder();
+              // create audio play and pause for unsaved recording
+              // set visualizer
```

Comment:
Line 245 and 246 are part of a full comment? (Let's make it a sentence.)

Reply:
I think 245 is meant to be a sentence that defines the section and 246 is meant to be a step in the implementation.

Reply:
Done.